aiida/restapi/translator/base.py

```diff
@@ -476,15 +422,19 @@ def get_formatted_result(self, label):
         if self._total_count > 0:
             for res in self.qbobj.dict():
                 tmp = res[label]
-                if self._default_user_projections:
-                    tmp['user_email'] = res['user']['email']
+
+                # Note: In code cleanup and design change, remove this node dependant part
```

Comment:
This note is supposed to stay here? Maybe instead remove it from here and open an issue? Otherwise we'll forget

Reply:
In redesign, this part anyways will be moved to NodeTranslator as baseTranslator won't be called for Node type. So I think there is no need to create an issue for this. Either we can remove the node completely or keep it for reference in code cleanup.